umh.docs.umh.app/content/en/docs/features/datainfrastructure/network-view.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Network Monitoring
+menuTitle: Network Monitoring
+description: |
+  Monitor all instances within the Unified Namespace at a glance.
+weight: 2000
+draft: true
+---
+
+The Network Monitoring feature of the United Manufacturing Hub visualizes the instance's
+network within the Unified Namespace.
+
+## When should I use it?
+
+Your infrastructure might consist of multiple UMH instances. This feature offers
+an intuitive view for monitoring the entire network of all instances and their
+network devices, allowing you to detect any issues within the infrastructure.
+
+## What can I do with it?
+
+The network view offers a graph where instances are usally grouped by the ISA-95 schema,
+these instances form a tree-like structure, which there can be multiple of, making it
```

Comment:
Does not sound good. My proposal:
The network view displays a graph in which instances are grouped according to the ISA-95 schema. These instances create a tree-like structure, facilitating the visualization of the entire network and the relationships within it. If there are instances without an allocation to the ISA-95 schema, they are shown on the side and not inserted into the rest of the graph structure.

Reply:
A typical user will only have one graph, and that is of his company. All other instances, like test instances, are then shown to the side.

Only in test cases, we will have multiple enterprises (like the demo mode)

Reply:
What I say is, the network view is a single graph, but this graph contains tree-like structures if the instances have defined locations that adhere to the ISA-95 model. So if there's a single enterprise (most likely), then there will be a single tree structure within the graph, while any instance that doesn't fit into that enteprise, is still part of the graph, but not part of the tree structure formed by the enterprise.